examples/sync/README.md

```diff
@@ -0,0 +1,110 @@
+# ADB Sync Example
```

Comment:
For consistency, always start with the crate name, then two badges, then the crate description: https://github.com/commonwarexyz/monorepo/blob/3ebb990819e78bc222554b75f19535ca351c8166/examples/chat/README.md?plain=1#L1-L8